service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/AbstractControllerTest.java

```diff
@@ -70,4 +95,21 @@ private AboutToStartOrSubmitCallbackResponse postEvent(String path, CaseDetails
             throw new RuntimeException(e);
         }
     }
+
+    private AboutToStartOrSubmitCallbackResponse postEvent(String path, byte[] data, int expectedStatus) {
+        try {
+            MvcResult response = mockMvc
+                .perform(post(path)
+                    .header("authorization", userAuthToken)
+                    .contentType(MediaType.APPLICATION_JSON)
+                    .content(data))
```

Comment:
We have got almost exact code as above. 

Could we maybe do: content.getBytes(StandardCharsets.UTF_8) 9as the content implementation is doing, and have this further simplified.

Reply:
done

Reply:
done